age--1.1.0.sql

```diff
@@ -3793,6 +3793,13 @@ IMMUTABLE
 PARALLEL SAFE
 AS 'MODULE_PATHNAME';
 
+CREATE FUNCTION ag_catalog.age_fact(variadic "any")
+RETURNS agtype
+LANGUAGE c
+IMMUTABLE
+PARALLEL SAFE
```

Comment:
Add the line "RETURNS NULL ON NULL INPUT"

Reply:
Done